src/gurt/dlog.c

```diff
@@ -762,12 +766,23 @@ int
 d_log_open(char *tag, int maxfac_hint, int default_mask, int stderr_mask,
 	   char *logfile, int flags)
 {
-	int	tagblen;
-	char	*newtag = NULL, *cp;
-	int	truncate = 0, rc;
-	char	*env;
-	char	*buffer = NULL;
-	uint64_t log_size = LOG_SIZE_DEF;
+	int		tagblen;
+	char		*newtag = NULL, *cp;
+	int		truncate = 0, rc;
+	char		*env;
+	char		*buffer = NULL;
+	uint64_t	log_size = LOG_SIZE_DEF;
+	int		pri;
+
+	mst.flush_pri = DLOG_WARN;
+
+	env = getenv(D_LOG_FLUSH_ENV);
```

Comment:
Is the environment  D_LOG_FLUSH_ENV  or D_LOG_FLUSH?  Readme does not have _ENV specified.

Reply:
Just saw #define in include file. Resolved.